theano/sandbox/cuda/opt.py

```diff
@@ -1052,13 +1061,13 @@ def local_gpu_subtensor(node):
         x = node.inputs[0]
         if (x.owner and
             isinstance(x.owner.op, HostFromGpu) and
-            x.dtype == "float32"):
+           x.dtype == "float32"):
```

Comment:
Does not align with parenthesis. I would prefer over-indenting all lines after the if.

```python
if (x.owner and x.dtype == "float32" and
        isinstance(x.owner.op, HostFromGpu)):
```

It doesn't align more with parenthesis but it's more clear that it belongs to it.